docs/examples/basic_tracer/resources.py

```diff
@@ -19,9 +19,10 @@
     BatchSpanProcessor,
     ConsoleSpanExporter,
 )
+from opentelemetry.semconv.resource import ResourceAttributes
```

Comment:
These are deprecated:

Use attributes defined in the :py:const:`opentelemetry.semconv.attributes` and `opentelemetry.semconv._incubating.attributes` modules instead.`